src/Equinor.ProCoSys.IPO.Infrastructure/Repositories/Fam/Queries/McPkgQuery.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Equinor.ProCoSys.IPO.Infrastructure.Repositories.Fam.Queries;
+
+public static class McPkgQuery
+{
+    public static string Query => @$"
+            SELECT
+	            m.Guid as ProCoSysGuid,

```

Comment:
Are we sure about this? Looking at the constructor of the mcPkg, seems to me the PcsGuid is the other one. I'm not even sure why we have this one.

Reply:
There can be several rows in the McPkgs table with the same Mc Package Guid, as the same Mc Package Guid can be connected to several invitations.
In order to to be able to send a delete message to FAM for we need a row unique Id, so FAM can remove that specific row.
That was the thinking behind the current logic of having a Guid column in addition to an McPkgGuid column.